src/assets/scss/_custom.scss

```diff
@@ -26,6 +26,7 @@ h1 small {
 .service {
 	padding: 20px;
 	box-shadow: 0 1px 4px rgba(0,0,0,0.2);
+	height: rem-calc(180);
```

Comment:
Forcing a fixed height on those boxes is likely to break on smaller screen sizes. You can probably remove this line,  now that you specify the min height on `.service p` instead.